qa/vagrant/src/test/resources/packaging/utils/packages.bash

```diff
@@ -95,6 +95,11 @@ verify_package_installation() {
     assert_file "$ESHOME/bin/elasticsearch-translog" f root root 755
     assert_file "$ESHOME/lib" d root root 755
     assert_file "$ESCONFIG" d root elasticsearch 2750
+    assert_file "$ESCONFIG/elasticsearch.keystore" f root elasticsearch 660
+
+    sudo -u elasticsearch "$ESHOME/bin/elasticsearch-keystore" list | grep "keystore.seed"
```

Comment:
Is this keystore list command meant to be checked? Is this run with `set -e` somewhere?

Reply:
All commands in the packaging tests are run with `set -e` (unless explicitly flipped, which we do not currently do anywhere).